google-cloud-pubsub/CHANGELOG.md

```diff
@@ -1,5 +1,18 @@
 # Release History
 
+### 1.0.0 / 2019-09-30
+
+### 1.0.0 / 2019-09-30
+
+ #### Features
```

Comment:
Shouldn't new features like this be released in a minor bump (`0.40.0`) that restarts the bake time for `1.0.0`?

Reply:
The Pub/Sub team has decided that these changes don't reset the bake-in.

Reply:
Similarly, they also decided the previous bug fixes don't reset the bake-in.

Reply:
ok